src/xrCore/xrCore.vcxproj

```diff
@@ -86,12 +87,12 @@
     </ClCompile>
     <Link>
       <SuppressStartupBanner>true</SuppressStartupBanner>
-      <AdditionalLibraryDirectories>$(DXSDK_DIR)Lib\x86;$(SolutionDir)Externals\lzo;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories>$(DXSDK_DIR)Lib\x86;$(SolutionDir)Externals\lzo;$(SolutionDir)Externals\tinyxml\$(Configuration)tinyxml;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
       <MapExports>false</MapExports>
       <SubSystem>Windows</SubSystem>
       <RandomizedBaseAddress>false</RandomizedBaseAddress>
       <TargetMachine>MachineX86</TargetMachine>
-      <AdditionalDependencies>cryptlib.lib;lzo2.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>cryptlib.lib;lzo2.lib;tinyxml.lib;tinyxml.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
This is not critical, but it's recommended to use project references instead of explicit pointing the lib name to the linker.
Also you have dublicated the lib name here.